src/Views/ListView/ListView.vala

```diff
@@ -385,41 +381,32 @@ public class Noise.ListView : ContentView, Gtk.Box {
         return status_text;
     }
 
-    private void view_search_func (string search, Gee.HashMap<int, Media> table, Gee.HashMap<int, Media> showing) {
+    private void view_search_func (string search, Gee.ArrayList<Media> table, Gee.ArrayList<Media> showing) {
```

Comment:
I'm not familiar with the code, but do you think there was any reason why `HashMaps` were chosen previously?

Reply:
No, I searched, but it looks like they were used like arrays. That's strange…